Assets/Scripts/Actors/ActorComponents/PlayerActorResources.cs

```diff
@@ -174,7 +174,7 @@ void OnTriggerEnter( Collider other )
 			PickupResource( resourceComponent.resourceData );
 
 			Destroy( other.gameObject );
-			WadeUtils.TempInstantiate( resourcePopPrefab, other.transform.position, Quaternion.identity, 1f );
+			SoundManager.instance.Play3DSoundAtPosition( resourcePopAudio, other.transform.position );
```

Comment:
I vote for wrapping up the public singleton interfaces with static methods that call private non static methods. e.g. static void Singleton::foo() { instance.privFoo(); }

That makes it easy to read the calling code and the implementation code. You lose the knowledge on the calling end that you're using a singleton but that shouldn't really matter.